unitary/examples/tictactoe/enums.py

```diff
@@ -14,6 +14,10 @@
 #
 import enum
 
+class GameMoves(enum.Enum):
+    EXIT = 0
```

Comment:
You should be able to do:

```
class GameMoves(enum.Enum):
    EXIT = 'exit'
    MAP = 'map'
    HELP = 'help'
```

Then, below you can do `GameMoves.EXIT.value` rather than doing string manipulation each time.

Reply:
This is what I had tried initially, but `GameMoves.EXIT.value` returns `0`, so it doesn't work. I did try using StrEnum as well, with similarly frustrating results.

Reply:
This is from an ipython I am running in my virtualenv:

```
In [2]: import enum

In [3]: class GameMoves(enum.Enum):
   ...:     EXIT = 'exit'
   ...:     MAP = 'map'
   ...:     HELP = 'help'
   ...: 

In [4]: GameMoves.EXIT.value
Out[4]: 'exit'
```

Reply:
I don't know what to say. It works when I run a python interpreter in the terminal and do exactly what you posted above. It doesn't work in the module. I added a `print(GameMoves.EXIT.value)` and it prints `0` even if I define the enum as you have above.

I added
```
        move = self.get_move()
        print('GameMoves.EXIT name and value')
        print(GameMoves.EXIT.name)
        print(GameMoves.EXIT.value)
```
to the `get_move()` function.

The output is:
```

        Welcome to quantum tic tac toe!
        Here is the board:
        
        a | b | c
        -----------
        d | e | f
        -----------
        g | h | i
        
Player X to move ("help" for help): help
GameMoves.EXIT name and value
EXIT
0
Catch error
```

That said, the tests pass with the enum using strings so...I'm guessing this is a version problem.